php/query_utils.php

```diff
@@ -462,6 +462,61 @@ function get_associated_survey_instances(
     return $ret;
 }
 
+/**
+ * Get a list of all surveys associated with a user_id. The
+ * search may be narrowed by further specifying a `course_code` and a `term`
+ */
```

Comment:
I'm actually not exactly sure how to filter survey by `$course_code` , because the only way to get course_code is by joining course_question_choices table, which could be null for a survey. Would it be fine to leave it out for now?